assets/nvidia.yaml

```diff
@@ -88,3 +88,25 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+- type: model
+  name: Nemotron 4
+  organization: Nvidia
+  description: Nemotron 4 is a 15-billion-parameter large multilingual language model trained on 8 trillion text tokens.
+  created_date: 2024-02-27
+  url: https://arxiv.org/pdf/2402.16819.pdf
+  model_card: none
+  modality: text; code, text
+  analysis: Evaluated on standard LLM benchmarks across a range of fields like reasoning, code generation, and mathematical skills.
+  size: 15B parameters (dense)
+  dependencies: []
+  training_emissions: unknown
+  training_time: 13 days
+  training_hardware: 384 DGX H100 nodes, each with 8 H100 80GB SXM5 GPUs
```

Comment:
maybe put the total number of gpus (i.e. what 384*8 is and then write this out after)